fix: ROOT-68: Add missing params discovered during SDK QA

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/16947791004)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/500
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16816222743

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16887221951

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-68/fixes'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16887221951

jombooth committed:
actions response

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16887390262

jombooth committed:
more accurate response shape

jombooth committed:
Merge branch 'fb-ROOT-68/fixes' of github.com:heartexlabs/label-studio into fb-ROOT-68/fixes

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16887759269

jombooth committed:
allow passing project id via query params instead

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16896176580

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-68/fixes'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16896176580

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16896577072

jombooth committed:
Merge branch 'fb-ROOT-68/fixes' of github.com:heartexlabs/label-studio into fb-ROOT-68/fixes

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16897015234

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16919213489

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-68/fixes'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16919213489

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16919832831

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-68/fixes'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16919832831

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16919955464

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16920708286)
> Successfully merged: Already up to date.

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16920697713

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16947008516)
> Successfully merged:  42 files changed, 2541 insertions(+), 552 deletions(-)

jombooth committed:
Merge branch 'develop' into 'fb-ROOT-68/fixes'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16947008516

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16947791004

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-68/fixes'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16947791004

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/16948710510
